src/js/hooks/productSupplier/form/useProductSupplierValidation.js

```diff
@@ -130,6 +130,7 @@ const useProductSupplierValidation = () => {
         .optional(),
       bidName: z
         .string()
+        .nullable()
```

Comment:
this can be either changed to `.nullish()` or removed at all, because you made a fallback for `?? undefined`